vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -173,6 +202,31 @@ void DevMenuManager::CreateAndShowUI() noexcept {
         }
       });
 
+#if defined(USE_HERMES)
+  m_samplingProfilerRevoker = devMenu.SamplingProfiler().Click(
+      winrt::auto_revoke,
+      [wkThis = weak_from_this()](
+          auto & /*sender*/, xaml::RoutedEventArgs const & /*args*/) noexcept -> winrt::fire_and_forget {
+        if (auto strongThis = wkThis.lock()) {
+          strongThis->Hide();
+          if (!Microsoft::ReactNative::HermesSamplingProfiler::IsStarted()) {
+            Microsoft::ReactNative::HermesSamplingProfiler::Start();
+          } else {
+            auto traceFilePath = co_await Microsoft::ReactNative::HermesSamplingProfiler::Stop();
```

Comment:
>auto [](http://example.com/codeflow?start=12&length=4)

(nit) is this string object? If so it would be nice to move it in .Post(...)

Reply:
Nikolai .. we want to the profiling to be stopped as soon as possible after use clicks .. Moving it into the UI dispatcher functor will introduce some delay .. Moreover I'd like to keep the stopping (which is the primary action) separate from the following steps such as copying the path to clipboard (auxiliary or optional steps) here .. Thanks!